src/main.c

```diff
@@ -159,6 +159,15 @@ int main(void) {
         while (1) {
         }
 
+    // Disable the watchdog, in case the application set it.
+#if defined(__SAMD51__)
```

Comment:
This is good but please change to `SAMD21` and `SAMD51` (see uses near line 83 in this file, and similar elsewhere.

Reply:
Done.